### PR TITLE
feat: backend view enrolled courses

### PR DESCRIPTION
**Pull request by yungjas**
- Get all approved enrollments of logged in user
- Returns class objects based on whether they are ongoing, upcoming or past
- Get class and course info in the following format:
  - course_name
  - class.id
  - class_name
  - progress
  - class_start_date
  - class_end_date
- To test using postman:
  - use post request and input a user token using json format (have to be logged in to printerLMS to get user token or just copy from database), i.e.
     `{
          "token": "asdfghjjkl"
     }`

### CODE REVIEW

File: backend/main.py [hunk 1]
```diff
@@ -218,6 +218,24 @@ def get_learner():
         return error.throw_error(type="Learner", message=str(e), status_code=400)
 
 
+@app.route("/api/enrollment/approved", methods=["POST"])
```
Comment (alphatrl) [inline backend/main.py:221]: enrolment

Comment (alphatrl) [inline backend/main.py:221]: enrolment

Comment (yungjas) [reply backend/main.py]: fixed

**Comment by alphatrl** [comment]
LGTM!

**Pull request merged**